docs/escape_support.md

```diff
@@ -73,6 +73,7 @@ brevity.
 | `CSI s`    | IMPLEMENTED |                                                   |
 | `CSI T`    | IMPLEMENTED |                                                   |
 | `CSI t`    | PARTIAL     | Only parameters `22` and `23` are supported       |
+|            |             | Rejected: `1`-`13`, `15`, `19`-`21`, `24`         |
```

Comment:
Maybe we can move rejected to a left column?

Reply:
Not a bad idea, sure!